gneiss/plot/_radial.py

```diff
@@ -0,0 +1,144 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2016--, gneiss development team.
+#
+# Distributed under the terms of the GPLv3 License.
+#
+# The full license is in the file COPYING.txt, distributed with this software.
+# ----------------------------------------------------------------------------
+import pandas as pd
+from gneiss.plot._dendrogram import UnrootedDendrogram
+
+try:
+    from bokeh.models.glyphs import Circle, Segment
+    from bokeh.models import ColumnDataSource, DataRange1d, Plot
+    from bokeh.models import HoverTool, BoxZoomTool, ResetTool
+except ImportError:
+    raise ImportWarning('Bokeh not installed. '
+                        '`radialplot` will not be available')
+
+
+def radialplot(tree, node_hue='node_hue', node_size='node_size',
+               node_alpha='node_alpha', edge_hue='edge_hue',
+               edge_alpha='edge_alpha', edge_width='edge_width',
+               figsize=(500, 500), **kwargs):
+    """ Plots unrooted radial tree.
+
+    Parameters
+    ----------
+    tree : instance of skbio.TreeNode
+       Input tree for plotting.
+    node_hue : str
```

Comment:
I think the usage of term "hue" here isn't right. You actually passed the entire RGB value to `hue`. But to be strict hue is a single number that should be used with saturation and brightness.